MoneySplitter.Win10/CustomControls/MobileUserControl.xaml

```diff
@@ -0,0 +1,129 @@
+﻿<UserControl
+    x:Class="MoneySplitter.Win10.CustomControls.MobileUserControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MoneySplitter.Win10.CustomControls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:extensions="using:MoneySplitter.Win10.Extensions"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+    <UserControl.Resources>
+        <Style TargetType="Button" x:Name="DefaultButtonStyle">
+            <Setter Property="Background" Value="Transparent" />
```

Comment:
You can remove default properties e.g. they are default. Leave only template and custom changed properties